web-local/src/lib/components/button/Switch.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+  export let checked = false;
+  export let id: string = undefined;
+</script>
+
+<input
+  bind:checked
+  class="
+  m-0
+  dark:transparent
+  checked:bg-gray-700 dark:checked:bg-gray-400
+"
+  role="switch"
+  type="checkbox"
+  {id}
+/>
+
+<style lang="postcss">
+  input {
+    @apply bg-gray-400;
+    --width: 22px;
+    --height: 12px;
+    --margin: 3px;
```

Comment:
nit: The placement of the circle and animation looks a bit weird for me in Firefox (105).
![Kapture 2022-10-23 at 12 22 11](https://user-images.githubusercontent.com/4402679/197378479-6dc9913d-e64f-4090-a1df-5da2d28e9bd5.gif)

Playing around with `--margin: 2px` it looks better. Although `3px` seems to look good in Chrome.